code/game/machinery/cryopod.dm

```diff
@@ -192,12 +192,20 @@
 		/obj/item/device/paicard,
 		/obj/item/weapon/gun,
 		/obj/item/weapon/pinpointer,
-		/obj/item/clothing/suit,
 		/obj/item/clothing/shoes/magboots,
 		/obj/item/areaeditor/blueprints,
 		/obj/item/clothing/head/helmet/space,
+		/obj/item/clothing/suit/space,
+		/obj/item/clothing/suit/armor/laserproof,
+		/obj/item/clothing/suit/armor/reactive,
+		/obj/item/weapon/tank,
```

Comment:
Not necessary. Plasma tanks aren't even checked for explicitly; it's the gas content of them.